code/datums/gamemode/factions/plague_mice.dm

```diff
@@ -0,0 +1,121 @@
+
+/datum/faction/plague_mice
+	name = PLAGUEMICE
+	ID = PLAGUEMICE
+	logo_state = "plague-logo"
+	hud_icons = list("plague-logo")
+
+	initroletype = /datum/role/plague_mouse
+	initial_role = PLAGUEMOUSE
+
+	roletype = /datum/role/plague_mouse
+	late_role = PLAGUEMOUSE
+
+	var/diseaseID = ""
+	var/datum/disease2/disease/bacteria/plague
+	var/turf/invasion
+
+
+/datum/faction/plague_mice/New()
+	..()
+	SetupDisease()
+	forgeObjectives()
+
+/datum/faction/plague_mice/HandleRecruitedRole(var/datum/role/R)
+	. = ..()
+	if (!plague)
+		SetupDisease()
+
+	var/mob/living/simple_animal/mouse/plague/M = R.antag.current
+	M.infect_disease2(plague,1, "Plague Mice")
+
+/datum/faction/plague_mice/OnPostSetup()
+	if (!plague || !invasion)
+		SetupDisease()
+
+	if (invasion)
+		for(var/datum/role/plague_mouse/M in members)
+			var/datum/mind/mouse_mind = M.antag
+			mouse_mind.current.forceMove(invasion)
+
+	spawn(1 MINUTES)
+		if(members.len > 0)
+			command_alert("A horde of black mice carriers of a dangerous bacteria have invaded the station. It is not clear how they got onboard, but they need to be either captured into small cages or exterminated, and any contaminated invividual is to cooperate with the medical staff for the preparation of a cure.", "The Black Plague is upon us!",1)
```

Comment:
This seems a bit obtuse. Could be summarized with "Hostile carrier vermin detected aboard the station. Contain or Pacify, before they begin to affect productivity.